p11kit/attribute.go

```diff
@@ -70,6 +70,16 @@ func (o *Object) matches(tmpl attribute) bool {
 	return false
 }
 
+// SetHandleID sets a new object handle id (CKA_OBJECT_HANDLE), allowing server
+// implementations to overwrite the randomly generated object handle id set in
+// constructor functions.
+//
+// Server implementations must ensure the specified id is unique amongst all
+// objects in a given session.
+func (o *Object) SetHandleID(id uint64) {
```

Comment:
nit: Just call this "SetID" then talk about handle ID in the comment?

Reply:
I'm a little hesitant as this may be confused for a function which sets `CKA_ID` or `CKA_OBJECT_ID` attributes if the comment is skimmed through.  That said if you feel strongly that the doc comment is sufficient I'll switch to `SetID`

Reply:
I've been trying to pretty aggressively avoid any knowledge of pkcs11 in the API. It's intended to be higher level.

For example, the doc comment might want to talk about when you'd set this rather than what it actually corresponds to.

```
SetID assigns a pre-determined identifier for an object, overriding the random one
generated by this package. This is required for some clients, such as Chrome, to
identify the same object over multiple sessions.

SetID should only be required when combined with Slot.GetObjects.
```

Reply:
Done